src/app/app.component.html

```diff
@@ -1,5 +1,7 @@
-<div>
-  {{title}}
+<div [class.light-theme]="lightTheme" class="container mat-app-background">
```

Comment:
chciałem dodać klase mat-app-background (dzięki ktorej background może sie zmieniać) do body do index.html, tylko wtedy nie dałbym rady zrobić tego przełącznika changeTheme() bo index.html nie ma pliku .ts
Chyba że w main.ts to mozna zrobic?

Reply:
Wrócimy do tego. Trzeba jeszcze kolory chartów zmieniać